diff --git a/ouster_client/src/client.cpp b/ouster_client/src/client.cpp
index af3d2cd..2d3ce47 100644
--- a/ouster_client/src/client.cpp
+++ b/ouster_client/src/client.cpp
@@ -54,7 +54,7 @@ Json::Value config_to_json(const sensor_config& config);
 namespace {
 
 // default udp receive buffer size on windows is very low -- use 256K
-const int RCVBUF_SIZE = 1024 * 1024;
+const int RCVBUF_SIZE = 16 * 1024 * 1024;
 
 int32_t get_sock_port(SOCKET sock_fd) {
     struct sockaddr_storage ss;
